test/sagemaker_tests/pytorch/training/resources/mnist/mnist.py

```diff
@@ -242,8 +241,6 @@ def assert_can_track_sagemaker_experiments():
 
 
 if __name__ == '__main__':
-    # test opencv
```

Comment:
if we remove this check, do we have a different way to test opencv?

Reply:
not for PT, but we do have for TF, [test_tensorflow_opencv](https://github.com/aws/deep-learning-containers/blob/3417251ebb9ad9888ece0fe2440469cd2f44aa23/test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py#L116)